tests/sources/test_manual_source.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+
+from telliot_feeds.dtypes.float_type import UnsignedFloatType
+from telliot_feeds.queries.abi_query import AbiQuery
+from telliot_feeds.sources.manual_sources.numeric_response import ManualSource
+
+
+class FakeQuery(AbiQuery):
+
+    value: str = ""
+
+    @property
+    def value_type(self) -> UnsignedFloatType:
+        """Data type returned for a DailyVolatility query. Returns a volitility index in decimals.
+
+        - abi_type: ufixed256x18 (18 decimals of precision)
+        - packed: false
+
+        """
+        return UnsignedFloatType(abi_type=self.value, packed=False)
+
+
+query = FakeQuery()
+source = ManualSource(query)
+
+
+@pytest.mark.asyncio
+async def test_one_number_response(monkeypatch):
+    query.value = "(fixed256x18)"
+    monkeypatch.setattr("builtins.input", lambda _: "[1]")
+    result, _ = await source.fetch_new_datapoint()
```

Comment:
In your tests, make it clearer what the user interface will look like by checking what the console output is going to be using `caplog` or something similar. For example, what is `telliot` going to say to me in the console before I enter `[1]` as a response?